### ISSUE TITLE
Stentura 400 Not Initializing

### ISSUE THREAD

**Issue by NewSteno** [openstenoproject/plover#947]
## Summary

Pretty much what it says here -- I've followed the connection steps, scan finds the com, but it won't initialize, and the red light keeps blinking.

**Comment by gdwaynewarner** [comment]
When you have a few minutes, go here:

http://www.cheapandsleazy.net/filez/stentura_200_400_srt_manual.pdf

No LCD display back then, so you had to rely on the lights to tell you what's what with the writer ... and, of course, to decode "Stenograph Binary," you need the manual.

**Comment by Greygal** [comment]
Make sure you are selecting the correct com port.  It's usually the highest-numbered one if you show more than one (for example, I see COM1 and COM4, the COM4 one is my Stentura).

Make sure you select "Stentura" as your machine in Plover.

Make sure you select the "Enable" radio button under the "Output" field on the main Plover screen (I won't admit how often I forget to select that).

Flashing red light usually means your memory is full, or your machine is in "student" mode.  It can also mean your battery needs charging, depending upon how fast the light is flashing.  It generally does NOT mean you are not connected - usually, if you are not connected, the red light is not turned on at all.

Go through the steps as listed here again.  They do work for both the 200SRT and 400SRT.

https://github.com/openstenoproject/plover/wiki/How-to-setup-and-use-Plover-with-a-Stentura-400SRT 

Hope this helps!

**Comment by gdwaynewarner** [comment]
When I was in court reporting school, I used a ProCAT Flash and digitalCAT, while most of my classmates were using the student version of Case CATalyst and the then-new Stentura 400 SRT.

One thing I would do that a couple of my classmates were amazed at was I could start digitalCAT, set up my writer -- and by that, I mean take it out of the case, put it together, and connect it to my laptop), initiate realtime in digitalCAT, and **THEN** turn on my writer.

They had to tap a couple of keys on their writer to get Case to recognize that the writer was there ... so maybe tap a couple keys during the initialization to see if that makes a difference.

I don't know if that is a requirement to get things going with Plover, as I wasn't impressed with Stenograph's offerings at the time, so I ignored my school's recommendations entirely, but my classmates seemed to have a **LOT** of issues with their writers and software.

So.  Tap a key or two during the initialization phase and see if that works.
